### ISSUE TITLE
UI Error at workflow screen

### ISSUE THREAD

**Issue by jzakotnik** [openkfw/TruBudget#1563]
In some cases (I believe if items are redacted), the workflow vertical line on the left is shifted:
![image](https://github.com/openkfw/TruBudget/assets/551599/17ff8033-506d-455f-ba02-09d6ec87a014)

**Comment by pmolnar-dev** [comment]
@jzakotnik Could you please describe the scenario when it does happen please so I can try to reproduce it? I see only this first line a bit off but the others are OK.
![Uploading Screenshot 2024-01-29 at 10.51.38.png…]()

**Comment by pmolnar-dev** [comment]
@jzakotnik Could you please describe the scenario when it does happen please so I can try to reproduce it? I see only this first line a bit off but the others are OK.
<img width="1027" alt="Screenshot 2024-01-29 at 10 51 38" src="https://github.com/openkfw/TruBudget/assets/55734106/f4a56780-373b-41a4-ace7-e1063e054c9e">

**Comment by jzakotnik** [comment]
My feeling is that it's related to the "redacted" workflow item, which you see in the screenshot of the bug, but not in yours. Could this be? I also couldn't reproduce the bug in workflows without redacted items.

**Comment by pmolnar-dev** [comment]
@jzakotnik Yes you are right I did manage to reproduce it that way. Just one more question. The "Redacted" text, should it be offset like you can see on screenshot below? Or it should be in line with other texts in other workflow items. Just want to know if it is intentional to have that offset
<img width="994" alt="Screenshot 2024-01-30 at 12 04 58" src="https://github.com/openkfw/TruBudget/assets/55734106/7dc0cb77-6a15-4711-910a-8a40e818fa5e">

**Comment by jzakotnik** [comment]
I don't have a strong opinion on this, whatever works best for you. Maybe without offset is better, also for a potentially responsive view in the future.
